src/interfaces/api/site.ts

```diff
@@ -99,10 +105,18 @@ export interface GetSiteResponse {
   banned: PersonViewSafe[];
   online: number;
   version: string;
-  my_user?: LocalUserSettingsView; // Gives back your local user and settings if logged in
+  my_user?: MyUserInfo; // Gives back your local user, settings, follows, and blocks if logged in
   federated_instances?: FederatedInstances;
 }
 
+export interface MyUserInfo {
+  local_user_view: LocalUserSettingsView;
+  follows: CommunityFollowerView[];
+  moderates: CommunityModeratorView[];
+  community_blocks: CommunityBlockView[];
+  person_blocks: PersonBlockView[];
+}
+
```

Comment:
`GetSite` now has an optional `MyUserInfo`, which contains your local user, subscribed comms, moderated comms, and blocks.

Reply:
I assume it is returned when a token is provided?